src/screens/ImageViewer/index.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-
+import {ActivityIndicator} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ActivityIndicator` from 'react-native' is a good addition as it will be used to show a loading indicator while images are being loaded. This improves the user experience by providing feedback during potentially long-loading operations.
```diff
+ import {ActivityIndicator} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->